src/v/datalake/cloud_uploader.cc

```diff
@@ -110,4 +110,22 @@ cloud_uploader::upload_data_file(
         co_return data_writer_error::cloud_io_error;
     }
 }
+ss::future<result<chunked_vector<coordinator::data_file>, data_writer_error>>
+cloud_uploader::upload_multiple(
```

Comment:
Is this where you mentioned you're missing a call? It feels like this should be called by the multiplexer, but it appears to be unused as of now. If that's intentionally missing, please call it out in the commit message